itl/plugins-contrib.d/network-components.conf

```diff
@@ -833,3 +833,231 @@ object CheckCommand "printer_health" {
 	vars.printer_health_hostname = "$check_address$"
 	vars.printer_health_mode = "supplies-status"
 }
+
+template CheckCommand "generic-thola-check-command" {
+	command = [ PluginContribDir + "/thola-client", "check" ]
+
+	arguments = {
+		"--target-api" = {
+			required = true
+			value = "$thola_api_address$"
+			description = "Address of the thola API"
+		}
+	}
+}
+
+template CheckCommand "generic-thola-device-check-command" {
+	import "generic-thola-check-command"
+	import "ipv4-or-ipv6"
+
+	arguments += {
+		"thola_device_address" = {
+			order = 0
+			required = true
+			skip_key = true
+			value = "$thola_device_address$"
+			description = "IP address of target device"
+		}
+		"--snmp-community" = {
+			value = "$thola_device_snmp_community$"
+			description = "SNMP Community of target device"
+		}
+		"--snmp-version" = {
+			value = "$thola_device_snmp_protocol$"
+			description = "SNMP Version of target device"
+		}
+	}
+
+	vars.thola_device_address = "$check_address$"
+}
+
+object CheckCommand "thola-cpu-load" {
+	import "generic-thola-device-check-command"
+
+	command += [ "cpu-load" ]
+
+	arguments += {
+		"--critical" = {
+			value = "$thola_cpu_load_critical$"
+			description = "Critical Threshold"
```

Comment:
Please re-consider whether it makes sense to start words in (all) descriptions upper-case (ex. sentence start, abbreviation, name, etc.).

Reply:
Btw. consider just copying them from the docs you've added.